examples/tipi-example/.tipi/deps

```diff
@@ -0,0 +1,5 @@
+{
+	"requires": {
+		"tipi-deps/tomlplusplus": { }
```

Comment:
Just as a note, this should be changed to `marzer/tomlplusplus` once the changes have been merged, currently pointing to our fork just to have the CI builds pass (otherwise this config would have to be more complex as it would require duplicating what's in `/.tipi/deps`, here's what it would look like:)

```json
{
	"requires": {
		"marzer/tomlplusplus": { 
			"requires": {
				"tipi-deps/Catch2@file://single_include": { "@": "v2.x" },
				"nlohmann/json": { "@": "v3.10.5" }
			}
		}
	}
}
```